test/cypress/support/commands.ts

```
@@ -404,6 +403,14 @@ const commands = {
             .click();
     },
 
+    clearDropdown: (fieldName: string) => {
+        cy.contains('.field', fieldName).within(() => {
+            if (Cypress.$(`.field:contains('${fieldName}') .delete.icon`).length > 0) {
+                cy.get('.delete.icon').click({ multiple: true });
+            }
```

What if the the `if` condition is not met? The command will just silently do nothing? Shouldn't it fail instead as all built-in Cypress commands do?
Also, why the `multiple` flag?

---

This function should clear dropdown when there is any item selected, so when it is clean we can't clear it again

`multiple` flag is for clicking multiple founded items by `get` command.

---

Is it meant to work with single selection dropdown (without `multiple` prop set) as well?
If not, I think we should name it `clearMultipleDropdown` or something like that to be more descriptive about the command purpopse.

---

I'm still not sure about the implementation of the command. If a command is intended to do something then I think it should either do it or fail if it's not possible to perform the action. It should also wait for a state change if it's not possible to perform the action at the time the command is executed. This implementation breaks this contract, thus:
- it's not in line with other Cypress commands, so it's behavior is unexpected
- it's error prone - since there is no wait for proper state (availability of delete icons) the result of command execution depends on exact timing
- it makes tests less robust - there are two possible execution paths (the `if`) while in any particular case only one is valid - in a robust test the command should be called in cases where there are items to be cleared, it shouldn't be called at all in cases when there is nothing selected

---

Ok @kubama let me remove this if statement.